test/e2e/infra/secondary_network/prepare_cluster.sh

```diff
@@ -0,0 +1,358 @@
+#!/bin/bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+pushd $THIS_DIR
+
+_usage="Usage: $0 [--setup] [--cleanup]
+                  [--base-interface <interface name>] [--sriov-vf-count <count>] --host-name <name>
+
+Deploy Antrea CNI and its native secondary network prerequisites on an existing/running K8s cluster control node.
```

Comment:
why do we keep mentioning that it needs to be a control Node? It could be any worker Node in a K8s cluster with the appropriate SRIOV configuration.

Reply:
Yes, But since we seek to have the SSH information of the cluster at the VMware test env, I kept it as control node. Also, with the current SRIOV support, we handle the secondary network b/w pods on the same nodes.